modules/schedule_module/test/TestPlan.md

```diff
@@ -0,0 +1,33 @@
+# Schedule Module Test Plan
+
+1. Create an Appointment [Automation Testing]
+Ensure that create does NOT work if one or more of the required feields are empty. 
+Ensure that the available sessions are correct for the candidate. 
+After successfully making an appointment, ensure the appointment has been added properly (table is properly populated)
+
+2. Edit an Appointment [Automation Testing]
+DCCID, PSCID, Session are non editable fields. 
+Ensure that any changes made show up in the table properly. 
+When user clicks "edit" but did not make any changes, an error swal will pop up indicating that no changes were made
+
+3. Delete an Appointment [Automation Testing]
+Delete by clicking the delete button. There should be an "Are you sure" message.
+
+4. Permissions [Automation Testing]
+Users should be able to see appointments across all sites, but should only be able to edit and delete appointments of candidates that belong to their site. 
```

Comment:
i'm still able to edit and delete appointments of candidates that do not belong to my site. i'm also able to create appointments for candidates not in my site. not sure if this is allowed

Reply:
I test it on IBIS, the user is able to create appointments for candidates that not belong to their site. I will update the test plan.